packages/oui-collapsible/src/collapsible.html

```diff
@@ -0,0 +1,13 @@
+<div class="oui-collapsible" data-ng-class="{'oui-collapsible_open':$ctrl.expanded}">
+    <div class="oui-collapsible__header" data-ng-click="$ctrl.toggle()">
```

Comment:
Accessibility issue: we have no clue when the header has the focus (tab navigation). 
We could add an outline like on the navbar for this :)

Reply:
This is done :)